fulp_modules/main_features/bloodsuckers/code/powers/brawn.dm

```diff
@@ -7,15 +7,15 @@
 	target_range = 1
 	power_activates_immediately = TRUE
 	must_be_capacitated = TRUE
-	can_be_immobilized = TRUE
+	can_use_w_immobilize = TRUE
 	bloodsucker_can_buy = TRUE
 	// Level Up
 	var/upgrade_canLocker = FALSE
 	var/upgrade_canDoor = FALSE
 
 /datum/action/bloodsucker/targeted/brawn/CheckCanUse(display_error)
 	. = ..()
-	if(!..(display_error)) // DEFAULT CHECKS
+	if(!.) // DEFAULT CHECKS
```

Comment:
display_error is meant to be there for all targeted powers afaik

Reply:
Doesnt ..() call the parent proc with all the same variables?